packages/apps/contracts/CommitRevealApp.sol

```diff
@@ -115,23 +115,13 @@ contract CommitRevealApp is CounterfactualApp {
     address[] memory to = new address[](1);
     uint256 player;
     if (state.stage == Stage.DONE) {
-      player = uint256(state.winner);
+      return abi.encodePacked(player);
```

Comment:
Can you import the enum symbol

Reply:
what would this change to after the import?

Reply:
The code needs to be as readable as possible so ideally you would be able to declare `player` variable using the enum.

Reply:
ah

Reply:
hmm isn't it a bit strange to have players be represented by an enum with value `SEND_TO_ADDR_ONE`?

Reply:
Perhaps the name is bad; should try to find a better name

Reply:
I will close this thread unless a better name is proposed for consideration